README.md

```diff
@@ -113,14 +113,25 @@ a PR to add it. Feel free to discuss in our Discord channel as well.
     - No, the source code will be executed on your machine. Any call to LLM APIs will be directly executed from your machine without any forwarding.
 
 2. Does `prompttools` store my API keys or LLM inputs and outputs to a server?
-    - No, all data stay on your local machine. No metrics, telemetry, or usage data are collected. As a result,
-      we would love to hear direct feedback from you. Please open an issue or join our Discord.
+    - No, all of those data stay on your local machine.
```

Comment:
Might be worth clarifying that we don't collect any PII (personally identifiable information)